### ISSUE TITLE
Reload command

### ISSUE THREAD

**Issue by xen0l** [erigones/Ludolph#22]
Ludolph should support reload command, which will restart him. Optionally, this could be combined with upgrade. The implementation should be simple:

1) Ludolph receives reload command and correctly exits
2) Serverside component will detect it and bring him up. This can be any service manager (monit, systemd, SMF etc)
3) Before starting ludolph, the startup script should try to update Ludolph's source code

**Comment by ricco386** [comment]
I have added command shutdown which can only admin execute and it will stop Ludolph (support roster announcement).

We should create wiki page how to configure monit, systemd, SMF to bring Ludolph up automatically. E.g., we can pre-prepare configuratin files similar to init.d scripts so user can easily reuse them...

**Comment by ricco386** [comment]
It would be nice to have SMF manifest, monit configration, however there is no way to update without have access to OS, and if user has access to Ludolph on OS he can use reload command, which does this thing.

There was a point that it is bad practice to stop program and rely to other stuff to start it again, not sure if this function should be kept in Ludolph as it is potentially dangerous.
